examples/express-minimal/package.json

```diff
@@ -2,12 +2,14 @@
   "name": "@examples/express-minimal",
   "version": "10.12.0",
   "private": true,
+  "type": "module",
   "scripts": {
     "dev:server": "tsx watch src/server",
     "dev:client": "wait-port 3000 && tsx watch src/client",
     "dev": "run-p dev:* --print-label",
     "lint": "eslint --ext \".js,.ts,.tsx\" --report-unused-disable-directives src",
-    "build": "tsc",
+    "type-check": "tsc",
+    "build": "pnpm type-check && esbuild src/server.ts src/client.ts --bundle --packages=external --platform=node --format=esm --outdir=dist --sourcemap",
```

Comment:
Instead of putting type-check here, we can run it as a separate step in ci to pralleöize it